lib/SILOptimizer/Differentiation/VJPEmitter.cpp

```diff
@@ -155,32 +155,18 @@ SILFunction *VJPEmitter::createEmptyPullback() {
   auto origParams = origTy->getParameters();
   auto indices = witness->getSILAutoDiffIndices();
 
-  // Add pullback parameter for the seed.
-  Optional<SILParameterInfo> inoutParam;
-  bool isWrtInoutParam = false;
+  // Add pullback parameters based on original result indices.
```

Comment:
Note: the pullback type calculation logic here in `VJPEmitter::createEmptyPullback()` duplicates `SILFunctionType::getAutoDiffDerivativeFunctionType`. It should be deduped, I'll look into it soon.